src/common/Timer.js

```diff
@@ -0,0 +1,20 @@
+class Timer {
```

Comment:
heh ;) I think this warrants it's own comments now in this file...which could subsequently change here...

My hope is to start collectively making our shared inventions available, to aid in making our work easier to share :)

this is a singularly consumed/piece wise block, not a library consumption. It's also evident that consuming React itself makes our entire solution susceptible to the Facebook BSD+patents license. 😸